src/entrypoints/json-stringify.d.ts

```diff
@@ -0,0 +1,24 @@
+interface JSON {
+  /**
+   * Converts a JavaScript value to a JavaScript Object Notation (JSON) string.
+   * @param value A JavaScript value, usually an object or array, to be converted.
+   * @param replacer A function that transforms the results.
+   * @param space Adds indentation, white space, and line break characters to the return-value JSON text to make it easier to read.
+   */
+  stringify<T = any>(
```

Comment:
BTW, is it useless to have the `= any` here? I'm just realizing I probably can remove this.